test/fixtures.yml

```diff
@@ -854,9 +854,9 @@ plugnpay:
   password: PASSWORD
 
 priority:
-  key: SANDBOX_KEY
-  secret: SECRET
-  merchant_id: MERCHANT_ID
+  # key: SANDBOX_KEY
+  # secret: SECRET
+  # merchant_id: MERCHANT_ID
```

Comment:
Nit: these placeholder values don't need to be commented-out.

Their presence in this file is meant to communicate what credentials are needed to access the gateway, and my worry with commenting them out is that it communicates something different, like maybe that they are obsolete or unnecessary for this particular gateway.

Reply:
This is a public repository. We have supplied credential via email for security

Reply:
Will email you the credentials.

Reply:
@priorityspreedly I apologize for the confusion here. We don't need you to share your credentials with us.

What I'm saying here is that we can just leave the code in this file as is, without commenting these fields out. Like this:
```
priority:
  key: SANDBOX_KEY
  secret: SECRET
  merchant_id: MERCHANT_ID
```

Reply:
Removed # from priority: credentials.